client/src/client.rs

```diff
@@ -394,9 +395,9 @@ where
     }
 
     /// Get connected peers.
-    pub fn get_peers(&self) -> Result<HashSet<SocketAddr>, handle::Error> {
+    pub fn get_peers(&self, services: Service) -> Result<HashSet<SocketAddr>, handle::Error> {
```

Comment:
As mentioned in the comment below, this should take `Into<ServiceFlags>`, so that either of the two types can be passed.

Reply:
Done